_emails/us/tennesee/clarksville.md

```diff
@@ -0,0 +1,42 @@
+---
+title: Clarksville
+permalink: "/clarksville"
+name: Letter to Mayor and Elected Officials
+city: Clarksville
+state: TN
+recipients:
+- joe.pitts@cityofclarksville.com
+- ward1@cityofclarksville.com
+- ward2@cityofclarksville.com
+- ward3@cityofclarksville.com
+- ward4@cityofclarksville.com
+- ward5@cityofclarksville.com
+- ward6@cityofclarksville.com
+- ward7@cityofclarksville.com
+- ward8@cityofclarksville.com
+- ward9@cityofclarksville.com
+- ward10@cityofclarksville.com
+- ward11@cityofclarksville.com
+- ward12@cityofclarksville.com
+subject:
+body: |-
+  Dear Mayor Pitts:
```

Comment:
Can we add language here to include the other recipients?

Reply:
What should the language be? I haven't heard of "wards" until now.

Reply:
Changed the salutation to "Dear Mayor Pitts _and Clarksville City Council Members_" based on [this](https://www.cityofclarksville.com/522/City-Council).